starcli/search.py

```diff
@@ -284,3 +237,20 @@ def search_github_trending(
     if order == "asc":
         return sorted(repositories, key=lambda repo: repo["stargazers_count"])
     return sorted(repositories, key=lambda repo: repo["stargazers_count"], reverse=True)
+
+
+def convert_gtrending_repo_to_repo(gtrending_repo):
```

Comment:
The function name is a bit long, how about `convert_repo_dict` or something? Totally up to you. Also It would be good if you could move this function declaration to before `search_github_trending` is defined, so we know it's a helper function

Reply:
okay, I'll rename `convert_repo_dict` ! Thanks.